girder/events.py

```diff
@@ -70,7 +53,6 @@ class Event(object):
         'currentHandlerName'
     )
 
-    @_deprecatedAsync
     def __init__(self, name, info, async_=False):
```

Comment:
We should just call this parameter `asynchronous`. `async_` is arbitrary and doesn't follow Girder's naming conventions.

Other upstream libraries have also used `asynchronous`:
* https://github.com/celery/celery/pull/4863
* https://github.com/celery/kombu/pull/839